index.html

```diff
@@ -29,6 +29,14 @@ <h2>Experience</h2>
             </ul>
         </section>
 
+        <section id="skills">
+            <h2>Skills</h2>
+            <ul>
+
```

Comment:
lgtm, is there a reason for you to fill in a blank space for design or for the code efficiency?